### DISCUSSION TITLE
Inverter status "online/offline"

### DISCUSSION THREAD

**Discussion by dk8pn** [DennisOSRM/hms-mqtt-publisher#89] [Ideas]
Would it make sense to publish the inverter status beside the electrical data? The only indicator that i have now, is that the time stamp of actualization stops.

**Comment by DennisOSRM** [comment]
Good idea. Any suggestions for a good topic name?

**Reply by dk8pn** [reply]
An idea is "connectivity".

I also took "DTU_SN" and "efficiency" from the "solar" json string. Both are not part of the "hms800wt2" MQTT topics?
